controller/array_action/array_mediator_ds8k.py

```diff
@@ -566,22 +566,20 @@ def create_snapshot(self, name, volume_name, pool_id=None):
         logger.info("finished creating snapshot '{0}' from volume '{1}'".format(name, volume_name))
         return self._generate_snapshot_response(target_api_volume, volume_name)
 
-    def _delete_flashcopy(self, flascopy_id):
+    def _delete_flashcopy(self, flashcopy_id):
         try:
-            self.client.delete_flashcopy(flascopy_id)
-        except exceptions.NotFound:
-            raise array_errors.VolumeNotFoundError(flascopy_id)
+            self.client.delete_flashcopy(flashcopy_id)
         except exceptions.ClientException as ex:
             logger.error(
                 "Failed to delete flascopy {} on array {}, reason is: {}".format(
```

Comment:
if you're already fixing `flascopy` to `flashcopy`, could you do it everywhere relevant?

Reply:
lol

Reply:
done